### PR TITLE
Review

### PR DESCRIPTION
**Pull request by travisbrown**

### CODE REVIEW

File: src/main/scala/io/github/finagle/example/petstore/PetstoreServer.scala [hunk 1]
```diff
@@ -48,11 +55,14 @@ class PetstoreServer extends HttpServer {
       override protected def configure(): Unit = {
 //        bind[PetstoreDb].in[Singleton]
```
Comment (travisbrown) [inline src/main/scala/io/github/finagle/example/petstore/PetstoreServer.scala]: Let's clear out all comments.

File: src/main/scala/io/github/finagle/example/petstore/controllerCaseClasses.scala [hunk 2]
```diff
@@ -0,0 +1,58 @@
+package io.github.finagle.example.petstore
+
+import com.twitter.finatra.request.{FormParam, RouteParam, QueryParam}
+
+/**
+ * Case class for the ID of a User, Pet, or Order
+ * @param id
```
Comment (travisbrown) [inline src/main/scala/io/github/finagle/example/petstore/controllerCaseClasses.scala]: No need for Scaladoc if there's nothing to say about the field.

File: src/test/scala/io/github/finagle/example/petstore/PetstoreFeatureTest.scala [hunk 3]
```diff
@@ -1,8 +1,14 @@
 package io.github.finagle.example.petstore
 
 import com.twitter.finagle.http.Status._
+import com.twitter.finagle.http.{RequestBuilder, Request, FileElement}
```
Comment (travisbrown) [inline src/test/scala/io/github/finagle/example/petstore/PetstoreFeatureTest.scala]: Let's alphabetize imports.

File: src/test/scala/io/github/finagle/example/petstore/PetstoreFeatureTest.scala [hunk 4]
```diff
@@ -38,21 +45,418 @@ class PetstoreFeatureTest extends FeatureTest {
...
+        params = Map("name" -> "Higgins", "status" -> "hungry"),
+        andExpect = NotFound
+      )
+    }
+
+    //addImage
+    "Allow the uploading of images for pets" in {
+      val imageDataStream = getClass.getResourceAsStream("/bear.jpg")
+      val imageByte = IOUtils.toByteArray(imageDataStream)
+      //                   Buf          Future[Buf]    Reader            InputStream
```
Comment (travisbrown) [inline src/test/scala/io/github/finagle/example/petstore/PetstoreFeatureTest.scala]: Don't really need this comment now.

**Comment by mimilei** [comment]
Issues should now be addressed!

**Pull request closed without merging**